lib/content-services/src/lib/dialogs/category-selector.dialog.html

```diff
@@ -0,0 +1,28 @@
+<h1 mat-dialog-title>
+  {{ 'CATEGORIES_MANAGEMENT.SELECT_EXISTING_CATEGORY' | translate }}
+</h1>
+
+<mat-dialog-content class="adf-dialog-content">
+    <adf-categories-management
+            (categoriesChange)="categories = $event"
+            [categoryNameControlVisible]="true"
+            [managementMode]="categoriesManagementMode"
+            [multiSelect]="multiSelect">
+    </adf-categories-management>
+</mat-dialog-content>
+<mat-dialog-actions align="end" class="acc-categories-dialog-actions">
```

Comment:
I don't see usages of acc-categories-dialog-actions class so can be removed. Also it has acc prefix which is incorrect as we are in ADF

Reply:
right, removed